test/functional/helpers/session.js

```diff
@@ -35,4 +66,10 @@ async function initDriver (caps, adbPort) {
   return driver;
 }
 
-export { initDriver };
+async function deleteSession () {
+  try {
+    await driver.quit();
```

Comment:
should we check if `driver` has value?

Reply:
Doesn't matter since we just ignore the error.